src/main/java/com/google/sps/placeGuide/repository/PlaceGuideRepository.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.placeGuide.repository;
+
+import org.jetbrains.annotations.Nullable;
+import com.google.sps.placeGuide.PlaceGuide;
+import java.util.List;
+
+/** Interface for handling place guide. */
+public interface PlaceGuideRepository {
+
+  public void savePlaceGuide(PlaceGuide placeGuide);
+
+  // @return all public {@code PlaceGuide}.
+  public List<PlaceGuide> getAllPlaceGuides();
```

Comment:
Can you please mark in the function name that it returns only public place guides?

Reply:
rename the function name to `getAllPublicPlaceGuides`

Reply:
changed to `getAllPublicPlaceGuides`